docs/dwarf-op.rst

```diff
@@ -1,158 +1,180 @@
+    It is recommended that you ``--show`` the selected dwarves before modifying
+    them.
+
+Options
+-------
+
+``--reset``
+    Forget which dwarves have been optimized. However, if you reload an existing
+    save, the optimization list will be reloaded.
+``--resetall``
+    Forget which dwarves have been optimized and remove the persisted
+    optimization data.
+``--show``
+    Lists the selected dwarves. Useful for previewing selected dwarves before
+    modifying them or looking up the migration wave number for a group.
+``--clean <value>``
+    Checks for skills with a rating of ``<value>`` and removes them from the
+    dwarf's skill list.
+``--clear``
+    Zeroes the skills and attributes of selected dwarves. No attributes, no
+    labors. Assigns ``DRUNK`` profession.
+``--reroll [inclusive]``
+    Clears attributes of selected dwarves, then rerolls that dwarf based on
+    their jobs. Run ``dwarf-op --list attrib_levels`` to see how stats are
+    distributed. If ``inclusive`` is specified, then attributes are not cleared,
+    but rather will only be changed if the current reroll is better. This
+    command ignores dwarves with jobs that are not listed in the ``jobs`` table.
+``--optimize``
+    Performs a job search for unoptimized dwarves. Run
+    ``dwarf-op --list job_distribution`` to see how jobs are distributed.
+``--applyjobs "[" <job> [<job> ...] "]"``
+    Applies the listed jobs to the selected dwarves. Run
+    ``dwarf-op --list jobs`` to see available jobs.
+``--applyprofessions "[" <profession> [<profession> ...] "]"``
+    Applies the listed professions to the selected dwarves. Run
+    ``dwarf-op --list professions`` to see available professions.
+``--applytypes "[" <profession> [<profession> ...] "]"``
+    Applies the listed types to the selected dwarves. Run
+    ``dwarf-op --list dwf_types`` to see available types.
+``--renamejob <name>``
+    Renames the selected dwarves' custom professions to the specified name.
+
+.. _dorf_tables:
+
+Data tables
+-----------
+
+The data tables that ``dwarf-op`` uses are described below. They can be
+inspected with ``dwarf-op --list <table name>``.
+
+``job_distributions``
+    Defines thresholds for each column of distributions. The columns must add up
+    to the values in the thresholds row for that column.  Every other row
+    references an entry in the ``jobs`` table.
+
+``attrib_levels``
+    Defines stat distributions, used for both physical and mental attributes.
+    Each level gives a probability of a dwarf randomly being assigned an
+    attribute level, and it provides a mean and standard deviation for the
+    attribute's value.
+
+    The ``p`` values describe the normal distribution of stats (each ``p`` value
+    has a sub-distribution, which makes the bell curve not so bell-shaped).
+    Labours do not follow the same stat system and are more uniformly random,
+    which are compensated for in the description of jobs/professions.
```

Comment:
```suggestion
    Defines stat distributions for both physical and mental attributes.
    Each level has a probability (p-value, or p) which indicates how likely
    a level will be used for a particular stat such as strength or spacial
    awareness. The levels range from incompetent to unbelievable (god-like)
    and are mostly inline with what the game uses already, but adds one
    level more than what the game uses to push the unbelievable even higher
    on average.

    In addition to a bell shaped p-value curve for the levels, there is
    additionally a standard deviation used to generate the value once a
    level has been selected this makes the bell curve not so bell shaped in
    the end. Labours do not follow the same stat system and are more uniformly
    random, which are compensated for in the description of jobs/professions.
```

This is how I would word it, but perhaps it doesn't need to be this precise or informative.